invokeai/backend/generator/txt2img2img.py

```diff
@@ -8,9 +8,9 @@
 import torch
 from diffusers.utils.logging import get_verbosity, set_verbosity, set_verbosity_error
 
-from ..models import PostprocessingSettings
+from ..stable_diffusion import PostprocessingSettings
 from .base import Generator
-from .diffusers_pipeline import (
+from ..stable_diffusion.diffusers_pipeline import (
     ConditioningData,
     StableDiffusionGeneratorPipeline,
     trim_to_multiple_of,
```

Comment:
This needs to be imported separately too.

`from ..stable_diffusion.diffusers_pipeline import trim_to_multiple_of`

Reply:
I just changed it have them all separate import, if that is fine?

Reply:
Should be alright.